config/initializers/twitter_regex.rb

```diff
@@ -0,0 +1,43 @@
+module Twitter
+  class Regex
+    require 'yaml'
```

Comment:
Wait, why are we requiring yaml here?

Reply:
oh, just forgetting to erase it after several tries in another way.